src/remote_tables/pushdown_visitor.rs

```diff
@@ -13,6 +14,16 @@ pub struct FilterPushdown<T: FilterPushdownVisitor> {
     pub sql_exprs: Vec<String>,
 }
 
+impl<T: FilterPushdownVisitor> FilterPushdown<T> {
+    // Intended to be used in the node post-visit phase, ensuring that SQL representation of inner
+    // nodes is on the stack.
+    fn pop_sql_expr(&mut self) -> String {
+        self.sql_exprs
+            .pop()
+            .unwrap_or_else(|| panic!("No SQL expression in the stack"))
```

Comment:
I think this could work too:

```suggestion
            .expect("No SQL expression in the stack")
```